core/src/main/java/com/datastax/oss/driver/internal/core/metadata/InitContactPointsRefresh.java

```diff
@@ -36,15 +35,15 @@
   }
 
   @Override
-  public Result compute(DefaultMetadata oldMetadata) {
+  public Result compute(DefaultMetadata oldMetadata, boolean tokenMapEnabled) {
```

Comment:
No, it's only here because it's inherited from the interface.
This refresh is applied before we're even connected to the first node, we don't know any tokens yet so the map can't be computed.